raiden/raiden_service.py

```diff
@@ -366,8 +378,12 @@ def get_new_events(self):
 
         return event_list
 
-    def open(self, token_address, partner_address,
-             settle_timeout=None, reveal_timeout=None):
+    def open(
+            self,
+            token_address,
+            partner_address,
+            settle_timeout=None,
+            reveal_timeout=None):
```

Comment:
The APIs that call non-constant smart contract functions need synchronization, this is required to avoid sending transactions for the same effect.

A user might make two or more HTTP requests with the same token and partner, since only first can succeed the additional ones will only burn gas.